Add section about local settings configuration. (#37829)
mistercrunch/Airflow: docs/apache-airflow/howto/set-config.rst

    that you run airflow components on is synchronized (for example using ntpd) otherwise you might get
    "forbidden" errors when the logs are accessed.

.. _set-config:configuring-local-settings:

Configuring local settings
==========================

> Some of Airflow configuration can be configured via local setting, because they require changes in the

Reviewer: ```suggestion
Some Airflow configuration is configured via local setting, because they require changes in the
```

> code that is executed when Airflow is initialized. Usually It is mentioned in the detailed documentation

Reviewer: ```suggestion
code that is executed when Airflow is initialized. Usually it is mentioned in the detailed documentation
```

> code that is executed when airflow is initialized. Usually It is mentioned in the detailed documentation

Reviewer: ```suggestion
code that is executed when Airflow is initialized. Usually It is mentioned in the detailed documentation
```

where you can configure such local settings - This is usually done in the ``airflow_local_settings.py`` file.

> You should create ``airflow_local_settings.py`` file and put in on ``$PYTHONPATH`` or
> to ``$AIRFLOW_HOME/config`` folder. (Airflow adds ``$AIRFLOW_HOME/config`` on ``PYTHONPATH`` when

Reviewer: ```suggestion
You should create a ``airflow_local_settings.py`` file and put it in a directory in ``sys.path`` or
in the ``$AIRFLOW_HOME/config`` folder. (Airflow adds ``$AIRFLOW_HOME/config`` to ``sys.path`` when
```

Reviewer: If we want a section to explain sys.path vs PYTHONPATH, we can, but I don't think we need it.

Author: Yep. I  just copied it from an existing explanation elsewhere - but this is cleaner.
